### PR TITLE
feat: add oracle price spread test

### PR DESCRIPTION
**Pull request by zarazan**
- Added oracle price spread test that sends a large spread of prices and verifies the correct median price is chosen

### CODE REVIEW

File: internal/modules/oracle/test.py [hunk 1]
```diff
@@ -121,6 +124,34 @@ def test_votes(self):
...
+            futures = []
+            for i in range(3):
+                futures.append(executor.submit(tx_send_prevote_and_vote, validators[i], exchange_rates[i]))
+            for future in concurrent.futures.as_completed(futures):
+                status, response = future.result()
+                self.assertTrue(status)
+        wait_for_next_voting_period(int(response['height']))
+        status, new_rates = query_exchange_rates()
+        self.assertTrue(status)
+        expected_rates = { 'ATOM': '3.0', 'JUNO': '2.5', 'UMEE': '0.05' }
```
Comment (adamewozniak) [inline internal/modules/oracle/test.py:151]: move this const to the top of the file please

File: internal/modules/oracle/tx.py [hunk 2]
```diff
@@ -7,6 +13,8 @@
 DEFAULT_GAS = env.DEFAULT_GAS
 DEFAULT_BROADCAST_MODE = "block"
 
+STATIC_SALT = "af8ed1e1f34ac1ac00014581cbc31f2f24480b09786ac83aabf2765dada87509"
+
```
Comment (adamewozniak) [inline internal/modules/oracle/tx.py:17]: can we use this from `oracle/test.py` so that we're not re-declaring here

**Review by adamewozniak — APPROVED** [review]
pre-approving 🥳 

@zarazan can you create an issue in this repo to make this test a bit more complex? I.e., we programmatically pick a set of prices, calculate the median, verify that the right median was accepted, and then loop through that 15 times

**Comment by zarazan** [comment]
> pre-approving 🥳 
> 
> @zarazan can you create an issue in this repo to make this test a bit more complex? I.e., we programmatically pick a set of prices, calculate the median, verify that the right median was accepted, and then loop through that 15 times

Good idea, I'll go ahead an make that change here.

File: internal/modules/oracle/test.py (hunk 1 above)
Comment (zarazan) [reply internal/modules/oracle/test.py:151]: I like your other idea of auto-generating these :)

**Comment by zarazan** [comment]
Merging this in even though all the tests are failing. It is not an issue with this PR. Attempting to fix it here: https://github.com/umee-network/qa-integration/pull/21

**Pull request merged**